internal/application/application.go

```diff
@@ -1,11 +1,16 @@
 package application
 
-import "ai-codereview/internal/domain"
+import (
+	"ai-codereview/internal/domain"
+
+	"github.com/google/go-github/v61/github"
+)
 
 type ICodeRepositoryProvider interface {
-	GetRepositoryName() string
+	GetEventName() string
 	GetRepository() string
-	CreateComment(comment string) error
+	GetPullRequestChanges() ([]github.CommitFile, error)
+	CreateComment(changedFiles []github.CommitFile) error
 }
```

Comment:
Code Review 2024-08-18 02:28:47 - New lines 7 to 8:

	GetPullRequestChanges() ([]github.CommitFile, error)
	CreateComment(changedFiles []github.CommitFile) error